index.gs

```diff
@@ -0,0 +1,62 @@
+function getTokenFromProperties() {
+  return PropertiesService.getUserProperties().getProperty("BOTIFY_API_TOKEN");
+}
+
+function init() {
+  SpreadsheetApp.getUi()
+    .createMenu("Botify Addon")
+    .addItem("Edit authentication        ", "promptForApiToken")
```

Comment:
What's all that space for?

Reply:
To make it look good ©️ 

No really, if I don't add the space the menu looks nasty...